content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -5,22 +5,28 @@ directory (`.dvc/`) with the
 [internal directories and files](#internal-directories-and-files) needed for DVC
 operation.
 
-Additionally, there are two special kind of files created by certain
+Additionally, there are a few special kind of files created by certain
 [DVC commands](/doc/command-reference):
 
-- Files ending with the `.dvc` extension are placeholders to version data files
+- Files ending with the `.dvc` extension are placeholders to track data files
   and directories. A <abbr>DVC project</abbr> usually has one
   [`.dvc` file](#dvc-files) per large data file or dataset directory being
   tracked.
 - The [`dvc.yaml` file](#dvcyaml-file) or _pipeline(s) file_ specifies stages
   that form the pipeline(s) of a project, and their connections (_dependency
   graph_ or DAG).
 
-Both use human-friendly YAML schemas, described below. We encourage you to get
-familiar with them so you may edit them freely, as needed. Both type of files
-should be versioned with Git (for Git-enabled <abbr>repositories</abbr>).
+  These typically come with a matching [`dvc.lock` file](#dvclock-file) to
+  record the pipeline state.
 
-## .dvc file
+Both `.dvc` files and `dvc.yaml` use human-friendly YAML schemas, described
```

Comment:
mention `dvc.lock` again? that it is auto generated?

Reply:
dvc.lock doesn't have a human-friendly YAML schema 😋  It's included in the following paragraph though: "All these should be versioned with Git..."